### DISCUSSION TITLE
Problem to conect whith ISolar_SMG_II_4KW_24V WiFi 

### DISCUSSION THREAD

**Discussion by Sensorv** [syssi/esphome-smg-ii#82] [General]
Hello.
I have a problem.
I can't connect and configure my inverter ISolar_SMG_II_4KW_24V WiFi to Home Assistant.
I use the file esp32-example.yaml to configure

All sensors and switches are inactive and off.
I use esp32 wroom 32.

Help me figure out the settings and connection
I attach еhe log and yaml file.

Thanks

[logs_esp_run (1).txt](https://github.com/user-attachments/files/18263146/logs_esp_run.1.txt)
[esp.yaml.txt](https://github.com/user-attachments/files/18263082/esp.yaml.txt)

![Screenshot_4](https://github.com/user-attachments/assets/7112e99b-a8e1-400e-9638-088bcbd0e755)
![Screenshot_3](https://github.com/user-attachments/assets/3ac2f004-6f48-4849-afce-658a369c38d7)
![5940cd06-019c-4249-b786-ee6f90577153](https://github.com/user-attachments/assets/128a4b68-910c-4d69-9152-1971da79efe7)
![20b300bb-cdda-4582-b692-c9c89edf8255](https://github.com/user-attachments/assets/e85f3054-4ec3-4604-9f91-2817c969656c)
![95c50e8d-ea70-4c54-9c97-42904570164a](https://github.com/user-attachments/assets/bc632b53-3859-4c1b-a70f-6fb71e7df939)
![7d680e6c-302c-47ea-93f4-2981a0ef18ab](https://github.com/user-attachments/assets/a4bf951c-9fad-49d5-b036-c022e0383aa7)
![ee3491a3-2e3f-40bb-bf42-992448668964](https://github.com/user-attachments/assets/219832d4-05d8-42e2-be4c-2ac787927005)

**Comment by syssi** [comment]
Let's ignore Home Assistant for now and let's focus on the ESPHome log. Please enable the debug output of the `uart` component:

https://github.com/syssi/esphome-smg-ii?tab=readme-ov-file#debugging

I guess there is just outgoing (`>>>`) traffic but no incoming (`<<<`). If this is the case we have to check your wiring as next step.

**Reply by Sensorv** [reply]
Thank you. I solved the problem. There was no contact on the signal line from the inverter. Now everything works.
